src/utils/rest-api.js

```diff
@@ -10,7 +10,13 @@ import { store } from '../redux/store';
 import ReconnectingWebSocket from 'reconnecting-websocket';
 import { ContingencyListType } from './elementType';
 import { CONTINGENCY_ENDPOINTS } from './constants-endpoints';
-import { ElementType } from '@gridsuite/commons-ui';
+import {
+  ElementType,
+  fetchEnv,
+  backendFetch,
+  backendFetchJson,
+  backendFetchText, getRequestParamFromList
```

Comment:
prettier let this pass?
```suggestion
  backendFetchText,
  getRequestParamFromList,
```